apps/hpc-ftsadmin/project.json

```
@@ -0,0 +1,106 @@
+{
+  "name": "hpc-ftsadmin",
+  "$schema": "../../node_modules/nx/schemas/project-schema.json",
+  "sourceRoot": "apps/hpc-ftsadmin/src",
+  "projectType": "application",
+  "targets": {
+    "build": {
+      "executor": "@nx/webpack:webpack",
+      "outputs": ["{options.outputPath}"],
+      "defaultConfiguration": "production",
+      "options": {
+        "compiler": "babel",
+        "outputPath": "dist/apps/hpc-ftsadmin",
+        "index": "apps/hpc-ftsadmin/src/index.html",
+        "baseHref": "/",
+        "main": "apps/hpc-ftsadmin/src/main.tsx",
+        "polyfills": "apps/hpc-ftsadmin/src/polyfills.ts",
+        "tsConfig": "apps/hpc-ftsadmin/tsconfig.app.json",
+        "assets": [
+          "apps/hpc-ftsadmin/src/favicon.ico",
+          "apps/hpc-ftsadmin/src/assets"
+        ],
+        "styles": [],
+        "scripts": [],
+        "webpackConfig": "@nx/react/plugins/webpack"
```

I realized here that our webpack config for FTS Admin isn't used at all. Not your fault, it is like that on `develop` now. It is defined like this in `workspace.json` currently, but it should be:
```suggestion
        "webpackConfig": "apps/hpc-ftsadmin/webpack.js"
```